app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/yeartodatelliability/YearToDateLiabilityController.scala

```diff
@@ -476,7 +486,7 @@ class YearToDateLiabilityController @Inject() (
         (answers, fillingOutReturn.draftReturn) match {
           case (calculatedAnswers: CalculatedYTDAnswers, draftReturn: DraftSingleDisposalReturn) =>
             withDisposalDate(draftReturn) { disposalDate =>
-              withEstimatedIncome(calculatedAnswers) { estimatedIncome =>
+              withEstimatedIncome(calculatedAnswers, isATrust(fillingOutReturn)) { estimatedIncome =>
```

Comment:
again, we should allow trusts to submit so you can guard this whole endpoint against trusts

Reply:
Done.